po/es.po

```diff
@@ -11361,27 +11363,27 @@ msgstr "proporción anchura/altura"
 
 #: ../src/iop/vignette.c:1185
 msgid "the radii scale of vignette for start of fall-off"
-msgstr "radio de la escala de viñeteado del principio del difuminado"
+msgstr "Dimensión del radio donde inicia el difuminado del efecto viñeta"
```

Comment:
A minor possible change  "Dimensión del radio donde **empieza** el difuminado del efecto viñeta"

Reply:
You have to use the module and then it makes perfect sense.